lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -222,6 +224,106 @@ def extrapolate(self, forward, n_estimate, extrapolation_length):
     def __len__(self):
         return len(self.coordinate_idx)
 
+    def msd(self, max_lag=None):
+        """Estimate the Mean Square Displacement (MSD) for various time lags.
+
+        The estimator for the MSD (rho) is defined as:
+
+            rho_n = (1 / (N-n)) sum_{i=1}^{N-n}(r_{i+n} - r_{i})^2
+
+        Note: This estimator leads to highly correlated estimates, which should not be treated
+        as independent measurements. See [1] for more information.
+
+        1) Michalet, X., & Berglund, A. J. (2012). Optimal diffusion coefficient estimation in
+        single-particle tracking. Physical Review E, 85(6), 061916.
+
+        Parameters
+        ----------
+        max_lag : int
+            Maximum lag to include.
+
+        Returns
+        -------
+        lag_time : array_like
+            array of lag times
+        msd : array_like
+            array of mean square distance estimates for the corresponding lag times
+        """
+        frame_lag, msd = calculate_msd(
+            np.array(self.time_idx, dtype=int),
+            np.array(self.coordinate),
+            max_lag if max_lag else len(self.time_idx),
```

Comment:
elsewhere we've used something like `max_lag or len(self.time_idx)`, is there any reason to use that style over the tertiary form or vice versa?

Reply:
Yeah, let's just use tertiary. I think the `or` is a bad habit I picked up in Lua.